components/index/InformationModal.tsx

```diff
@@ -1,28 +1,28 @@
-import { Modal } from "react-bootstrap";
-import {
-  GreenCloseButton,
-  CustomHeader,
-  CustomFooter,
-  CustomDescriptionBody,
-} from "../../styles/styledComponentModule";
+import {Modal} from "react-bootstrap";
+import {CustomDescriptionBody, CustomFooter, CustomHeader, GreenCloseButton,} from "../../styles/styledComponentModule";
 import AdFitModal from "../advertisement/adFitModal";
-import { informationModalAdID } from "../advertisement/ad-ids";
+import {informationModalAdID} from "../advertisement/ad-ids";
+import {Swiper, SwiperSlide} from "swiper/react";
+import {EffectFade, Navigation, Pagination} from "swiper/modules";
+import "swiper/css";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/css/effect-fade";
+import React, {useState} from "react";
+import styled from "styled-components";
+import {header_1, header_2, header_3, slide_1, slide_2, slide_3} from "./informationModaltext";
 
 const InformationModal = (props) => {
+  const [index, setIndex] = useState<number>(0);
```

Comment:
```suggestion
  const [index, setIndex] = useState(0);
```

타입 추론 기능이 있어 굳이 number, string 과 같은 타입은 지정해주지 않아도 된대용 ㅎㅎ